src/main/java/OldCode/Deadline.java

```diff
@@ -0,0 +1,14 @@
+package OldCode;
+
+public class Deadline extends Task{
+    private String when;
+    public Deadline(String description, String when) {
+        super(description);
+        this.when = when.substring(3);
```

Comment:
Consider using a more appropriate name (when is not a noun, compared to time or date)